src/CMakeLists.txt

```diff
@@ -62,7 +62,7 @@ SET(XRT_VERSION_MINOR 11)
 if (DEFINED ENV{XRT_VERSION_PATCH})
   SET(XRT_VERSION_PATCH $ENV{XRT_VERSION_PATCH})
 else(DEFINED $ENV{XRT_VERSION_PATCH})
-  SET(XRT_VERSION_PATCH 0)
+  SET(XRT_VERSION_PATCH 722)
```

Comment:
This change should be removed when sending PR to XRT repo

Reply:
Sure